### ISSUE TITLE
Apollo repeated crashes

### ISSUE THREAD

**Issue by sperdog** [christianselig/apollo-bugs#570]
Every time I try to submit a post, Apollo crashes. I’m only about to post comments and DM. Been happening since I started using Apollo two years ago. Just decided to finally try to submit bug report cause I thought it would be fixed eventually.

**Comment by christianselig** [comment]
Can you go to Settings tab > About > Crash Reports & Analytics > LONG PRESS on the top row and share your Crash ID with me?

**Comment by sperdog** [comment]
31EB47E8-D7F7-42E6-ACC0-2FF82360367B

**Comment by christianselig** [comment]
It looks like you're not inputting anything into the title field/input, can you confirm if that's the case?

**Comment by sperdog** [comment]
When I’m submitting posts? Some of the subreddits I try to post say not to put anything in the Title. Only to type in the text box.

**Comment by christianselig** [comment]
>When I’m submitting posts? Some of the subreddits I try to post say not to put anything in the Title. Only to type in the text box.

Can you give an example of a subreddit that says this?

**Comment by sperdog** [comment]
AskReddit, ELI5, PersonalFinance, ShowerThoughts, CasualConversation. Now that you mention it, I was able to submit a post in PhotoshopBattles with a title awhile back.

**Comment by christianselig** [comment]
Those subreddits all require a title, they say not to include a body.

**Comment by sperdog** [comment]
It said Text Only for me. I did view this rules for ELI5 and it does say to a title. But the other ones I didn’t see anything in the rules for title.

**Comment by christianselig** [comment]
AskReddit: Rule 1 - Questions must be clear and direct and may not use the body textbox
(Can't use body, must use title)

PersonalFinance: Doesn't say anything, lots of posts have both

ShowerThoughts: Rule 3 - Your entire thought must be in the title.
(Can't use body, must use title)

Those are just your first three examples, I think you're mixing up titles and bodies, haha. Reddit doesn't permit posts to be submitted in any subreddit without a title. I admittedly didn't realize people were trying this so Apollo doesn't handle it correctly, in the next build I'll have it prevent you from trying.

**Comment by sperdog** [comment]
Wow. I was definitely confusing ‘text only’ thinking we can only type in the text box rather to only use text. You think it is saying for people not to put images in the text box?

Thanks for your help Christian. That would have definitely been helpful all this time if I would have been prompted to put something in the Title rather than just crashing.

**Comment by christianselig** [comment]
Essentially there are three kinds of posts: images, links, and text. Titles are required for all of these. The text type is just for discussions, like a thought provoking question, and if the title is sufficient for getting your question/point across, you don't need to fill in the body.

Yeah it definitely shouldn't crash haha. I'll fix that and make it more clear that you need a title for the next update.

**Comment by christianselig** [comment]
Made this more clear in new version
